src/test/java/bdd/runner/CucumberTestRunner.java

```diff
@@ -0,0 +1,18 @@
+package bdd.runner;
+
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import io.cucumber.testng.CucumberOptions;
+
+@CucumberOptions(
+        plugin = {"pretty", "html:target/cucumber-reports",
+                "json:target/cucumber-reports/CucumberTests.json",
+                "testng:target/cucumber-reports/CucumberTests.xml",
+                "json:target/Cucumber.json",
```

Comment:
why you need lines 10-11 ? Duplication of lines 7 and 8

Reply:
Deleted, thank you for pointing.
Please see in the following commit.